docs/markdown/Reference-manual.md

```diff
@@ -2408,6 +2408,12 @@ page](Configuration.md) It has three methods:
 
 - `has(varname)`: returns `true` if the specified variable is set
 
+- `keys()`*(since 0.57.0)*: returns an alphabetically sorted array of keys of
```

Comment:
Note sure we should specify the ordering here, sounds more like an implementation detail. I fear this could open a can of worms if people expect Unicode ordering, which would be leaking python retail in our API.

Reply:
Got your point, I'll change the description to just "sorted", so it is mentioned that is sorted, but not further specified how.
```suggestion
- `keys()`*(since 0.57.0)*: returns a sorted array of keys of
```

Reply:
I would even not mention ordering at all, we don't mention ordering of dict neither that's a design decision to leave that undefined. As an implementation detail however I agree we need to make sure the order is stable across reconfigure.